### PR TITLE
Add test for repeated UUID invocation (w3c/sparql-12#102).

### PR DESCRIPTION
**Pull request by kasei**
Adds a test to cover issue raised in w3c/sparql-12#102.

### CODE REVIEW

**Comment by kasei** [comment]
I've had this sitting around for more than a year and just realized I submitted the PR to my fork instead of w3c/rdf-tests. Oops.

File: sparql11/data-sparql11/functions/manifest.ttl [hunk 1]
```diff
@@ -733,6 +733,18 @@
     mf:result  <uuid01.srx> ;
 	.
 
+:uuid02 rdf:type mf:QueryEvaluationTest ;
```
Comment (afs) [inline sparql11/data-sparql11/functions/manifest.ttl:736]: There isn't a "uuid02" in the  mf:entries list of tests.

Also and maybe related - I could not apply the manifest part as a patch.

**Comment by afs** [comment]
Test passes.

I had a problem with the manifest patch (got out of step?)

**Review by rubensworks — APPROVED** [review]
Looks good to me, test passes on my end.

File: sparql11/data-sparql11/functions/manifest.ttl (hunk 1 above)
Comment (kasei) [reply sparql11/data-sparql11/functions/manifest.ttl:737]: Added to the entries list. I'm not sure why the patch would have failed.

**Comment by kasei** [comment]
@afs I merged in recent changes from the gh-pages branch. I think it should patch cleanly now.

**Review by afs — APPROVED** [review]

**Review by gkellogg — APPROVED** [review]

**Pull request merged**